test/distributed/_pycute/test_coalesce.py

```diff
@@ -0,0 +1,99 @@
+# ruff: noqa: PGH004, G004, F403
```

Comment:
Lol you should double check underscore in the test directory doesn't cause pytest to stop picking it up haha

Reply:
wait do you mean _pytest won't get picked up? But I do run into CI failure before I use "from torch.testing._internal.common_utils import run_tests, TestCase"